devtools/config-doc-maven-plugin/src/main/java/io/quarkus/maven/config/doc/generator/AbstractFormatter.java

```diff
@@ -31,12 +31,12 @@ public String formatDescription(ConfigProperty configProperty) {
             return null;
         }
 
-        String description = javadoc(javadocElement.get());
+        String description = javadocElement.get().description();
```

Comment:
I have a javadoc link in one of my comments:

    See {@link FeedRefreshIntervalCalculator} for details.

Without the change, the javadoc tag was printed verbatim.
With the change, the link is correctly replaced with a markdown code block. Do you have another idea on how to transform it correctly? Now that I think about it, maybe I shouldn't use javadoc links in those comments altogether since they are intended for an audience that won't read source code.

Reply:
Don’t you end up with an Asciidoc link and not a Markdown one?
Where does it point exactly? Because I don’t see how it could find your Javadoc base URL?

I think we should support that and what I was wondering is if we should do it in the Maven plugin. But I’m not sure you have everything to do it.

Reply:
Oh and don't change your Javadoc for now, I'll see if I can improve things. It's easier if I have an example around.

Reply:
> Don’t you end up with an Asciidoc link and not a Markdown one?

Neither, I get a code block, both in asciidoc and in markdown:
![image](https://github.com/user-attachments/assets/47d2b529-7391-4839-b3e7-4206115cbddc)

Reply:
Ah OK, I see. You're lucky enough that the generated Asciidoc is also valid Markdown.

Could you reintroduce the `javadoc` methods you removed? I will fix it another way soon. Thanks!

Apart from that I think we're good to go, please ping me when done.

Reply:
Sure!